lib/matplotlib/axes/_axes.py

```diff
@@ -3109,29 +3109,34 @@ def pie(self, x, explode=None, labels=None, colors=None,
             A sequence of colors through which the pie chart will cycle.  If
             *None*, will use the colors in the currently active cycle.
 
+        hatch : str or list, default: None
```

Comment:
Sorry if I'm ignorant here, but shouldn't this go at the end of the parameter documentation so that parameters are documented in the order they're in the method signature?

Reply:
There is an earlier discussion about that in the thread. Basically: yes, but if it makes sense to group some odd arguments that may outweigh it.

Reply:
Fab, I'm happy as long some thought has been put into it 😃